src/App.tsx

```diff
@@ -16,46 +16,90 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortType, setSortType] = useState('none');
+  const [isReversed, setDirection] = useState(false);
+
+  const reset = () => {
```

Comment:
let's rename the function according to standards: `on` + `Action`.
please check all cases
```suggestion
  const onReset = () => {
```